env.sh

```diff
@@ -1,3 +1,10 @@
 #!/bin/bash
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
 
 . `pwd`/env/conda/bin/activate fpga-interchange
```

Comment:
FYI - You should never use backticks, always use `$(xxx)`

Reply:
All right, I'll fix that with a follow-up PR, thanks